src/api/index.ts

```diff
@@ -36,14 +37,45 @@ export const Responses = {
   }
 }
 
+export type SIGNALK_API_ID =
+  | 'resources'
+  | 'course'
+  | 'history'
+  | 'autopilot'
+  | 'anchor'
+  | 'logbook'
+
+export const isSignalKApi = (id: SIGNALK_API_ID): boolean => {
```

Comment:
I think parameter should be `string` -  type SIGNALK_API_ID says it is a Signal K Api Id, so why would you call it with only those?